app/views/candidate_interface/offer_dashboard/view_reference.html.erb

```diff
@@ -46,7 +46,7 @@
     <div class="govuk-button-group">
       <%= render(CandidateInterface::ReferenceSendReminderComponent.new(@reference)) %>
 
-      <%= govuk_link_to candidate_interface_references_confirm_cancel_reference_path(@reference.id) do %>
+      <%= govuk_link_to candidate_interface_references_confirm_cancel_reference_path('request-reference', @reference.id) do %>
```

Comment:
Can we discuss if using a string inquirer is better for all those or call the constant of the suggestion abstractions to centralize those values somewhere?

Reply:
Probably call that constant somehow.